kuma/wiki/views/document.py

```diff
@@ -875,8 +874,8 @@ def react_document(request, document_slug, document_locale):
 
     # Bundle it all up and, finally, return.
     context = {
-        'document_api_data': base64.b64encode(
-            json.dumps(document_api_data(doc))),
+        'document_api_data': document_api_data(doc),
```

Comment:
Yay!

Reply:
Later, when we want to pre-render the SSR files to S3, I suppose we'll get the document API data directly from the pre-rendered data on S3 (e.g., `https://<s3-website-url>/api/v1/doc/<locale>/<slug>`)?